test/sql/types/timestamp/test_timestamp_types.test

```diff
@@ -0,0 +1,109 @@
+# name: test/sql/types/timestamp/test_timestamp_types.test
+# description: Test TIMESTAMP types
+# group: [timestamp]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
```

Comment:
I would like to see some additional tests:
* Storage tests (test/sql/storage/types/test_timestamp_storage.test)
* GROUP BY on each of these timestamp types
* Joins on the timestamps
* Sorting on the timestamps
* Comparisons between all the different timestamp types (e.g. TIMESTAMP = TIMESTAMP_MS, etc)